config/schema.json

```diff
@@ -238,6 +238,10 @@
           "description": "Object value that specifies an opaque JSON configuration. The JSON is stored and returned along with the service instance when called from the API.",
           "type": ["object", "null"]
         },
+        "localServiceAddress": {
```

Comment:
Can you add some tests for this new field? Existing tests for parsing the schema can be found here https://github.com/hashicorp/consul-ecs/blob/main/config/validate_test.go

Reply:
@Ganeshrockz Thanks and done! I added them to the existing `expectedExtensiveConfig` based tests since the omitted value should be covered by other tests. Please let me know if you would like me to rework that into its own test. Thanks!

Reply:
This looks good. Thanks for adding those tests.